src/autopas/containers/ParticleContainer.h

```diff
@@ -38,7 +38,7 @@ class ParticleContainer : public ParticleContainerInterface<ParticleCell> {
       : _cells(), _boxMin(boxMin), _boxMax(boxMax), _cutoff(cutoff), _skin(skin) {}
 
   /**
-   * destructor of ParticleContainer
+   * Destructor of ParticleContainer
```

Comment:
```suggestion
   * Destructor of ParticleContainer.
```
while we are at it :P